.cirrus.yml

```diff
@@ -182,7 +220,9 @@ report_stageone_task:
             dnf -y install python3-pip ethtool
             dnf -y install ./sos_${BUILD_NAME}.rpm
         fi
-        pip3 install avocado-framework==94.0
+        PIP_EXTRA=""
+        [[ $(pip3 install --help | grep break-system) ]] && PIP_EXTRA="--break-system-packages"
```

Comment:
This is redundant to add on line 133 since Ubuntu's `pip` always has `--break-system-packages` option, I guess?

Reply:
I have tested all this, and it's not the case. 18.04, 20.04 and 22.04 don't have this option, and therefore would error out. This is now an option only in the new python3, which is in 23.10 that we're doing the test for the deb package. I haven't fully tested which pip3 version this was introduced

Reply:
I meant it in opposite way :)  Does it make sense to add the `PIP_EXTRA` trick to line 133 as well?

Reply:
Ah, I get it

We are only doing the deb testing on the latest Ubuntu release, as that is where we'll see problems. So in this scenario we don't need it as we'll always need that option there with newer releases.

I'm sure we'll see the same problem when rhel 10 and newer Fedora comes out too